fastapi/security/jwt.py

```diff
@@ -102,22 +97,22 @@ def _generate_payload(
             "subject": subject.copy(),  # main subject
             "exp": now + expires_delta,  # expire time
             "iat": now,  # creation time
-            "jti": str(uuid.uuid1()),  # uuid
+            "jti": str(uuid1()),  # uuid
         }
 
         return to_encode
 
     def _get_token(
         self,
-        bearer: Optional[HTTPBearer],
-        cookie: Optional[APIKeyCookie],
+        bearer: Optional[HTTPBearer] = None,
+        cookie: Optional[APIKeyCookie] = None,
     ) -> Optional[str]:
-        token = None
-
-        if cookie is not None:
-            token = str(cookie)
-        if bearer is not None:
+        if bearer:
             token = str(bearer.credentials)  # type: ignore
+        else:
+            if cookie:
+                cookie = str(cookie)
+            token = cookie
```

Comment:
After this correction, the logical order of `places` prioritization is violated.  
It seems to me that it is more important when providing a Bearer token to use it, since it is specified more explicitly than cookies.  
Perhaps then the best way is the following
```
if bearer:
    return str(bearer.credentials)  # type: ignore
if cookie:
    return str(cookie)
return None
```